package.json

```diff
@@ -9,22 +9,22 @@
   "scripts": {
     "acceptance": "scripts/acceptance.sh",
     "acceptance-record": "scripts/acceptance_record.sh",
-    "assets": "scripts/assets.sh",
-    "build": "yarn webpack",
+    "assets": "NODE_ENV=production yarn build",
+    "build": "yarn clean && yarn webpack",
     "bundle-report": "NODE_ENV=production ANALYZE_BUNDLE=true yarn build",
     "bundle-report:dev": "NODE_ENV=development ANALYZE_BUNDLE=true yarn build",
-    "clean": "rm -rf public/assets",
+    "clean": "rm -rf public/assets && mkdir -p public/assets",
     "mocha": "scripts/mocha.sh",
     "precommit": "lint-staged",
     "prepare": "patch-package",
     "prepush": "yarn type-check",
     "prettier-project": "yarn run prettier-write 'src/**/*.{ts,tsx,js,jsx}'",
     "prettier-write": "yarn run prettier --write",
-    "publish_assets": "scripts/publish_assets.sh",
+    "publish-assets": "yarn assets && node scripts/uploadToS3.js",
     "relay": "node scripts/relayCompiler.js",
     "sentry": "scripts/sentry.sh",
     "start": "scripts/start.sh",
-    "start:prod": "NODE_ENV=production node -r dotenv/config --max_old_space_size=1024 ./src --verbose --debugProd",
+    "start:prod": "NODE_ENV=production node -r dotenv/config --max_old_space_size=4096 ./src --verbose --debugProd",
```

Comment:
If we're increasing the mem cap of force we need to make sure that's reflected in our infra. cc @izakp

Reply:
I guess this isn't actually doing what I thought. Is this just for debugging prod?

Reply:
Yup, for running locally. (I could update the script name.)